refactor/urls.js

```diff
@@ -1,16 +1,20 @@
-// Refactor this code
-
 function getURL(dev = false, tasks = {}) {
-    let url = dev
-        ? `/tasks?status=ACTIVE&dev=true&size=20`
-        : '/tasks';
+  let baseURL = "/tasks";
+  const params = new URLSearchParams();
+
+  if (dev) {
+    params.append("status", "ACTIVE");
+    params.append("dev", true);
+    params.append("size", 20);
+  }
+
+  if (tasks.nextTasks) {
+    params.append("hasNext", "true");
+  }
 
-    if (tasks.nextTasks) {
-        url += '?hasNext=true';
-    }
+  if (tasks.prevTasks) {
+    params.append("hasPrev", "true");
+  }
 
-    if (tasks.prevTasks) {
-        url = '/tasks?hasPrev=true';
-    }
-    return { url };
+  return { url: `${baseURL}?${params.toString()}` };
```

Comment:
What happens when there are no queryParams?

Reply:
Handled this case and committed my code again.
  const queryString = params.toString();
  return { url: queryString ? `${baseURL}?${queryString}` : baseURL };